pkgs/by-name/gp/gpt4all/package.nix

```diff
@@ -56,6 +67,7 @@ stdenv.mkDerivation (finalAttrs: {
     "-DKOMPUTE_OPT_USE_BUILT_IN_VULKAN_HEADER=OFF"
     "-DKOMPUTE_OPT_DISABLE_VULKAN_VERSION_CHECK=ON"
     "-DKOMPUTE_OPT_USE_BUILT_IN_FMT=OFF"
+    "-DLOCAL_EMBEDDING_MODEL_PATH=${finalAttrs.embed_model}"
```

Comment:
This line is probably unnecessary with the CMakeLists.txt changes.

Reply:
Removed.